src/services/AuthorizationRequestProcessor.ts

```diff
@@ -49,6 +49,7 @@ export class AuthorizationRequestProcessor {
 		if (session != null) {
 			if (session.expiryDate < absoluteTimeNow()) {
 				this.logger.error("Session has expired", { messageCode: MessageCodes.EXPIRED_SESSION });
+				this.logger.appendKeys({ sessionId });
```

Comment:
can this be added before if check pls

Reply:
Is that the if on line 49 or the if on 50?

Reply:
before line 49 incase we got logs saying no session with this id found